### DISCUSSION TITLE
No fields in c.coverage-audit and  c.console index pattern

### DISCUSSION THREAD

**Discussion by josip1001** [canarytrace/documentation#186] [Synthetic]
Hi Radim, 
we are using predefined Canarytrace dashboards in Kibana
We have some issues to display results:

1.) Canarytrace Errors dashboard ,widgets: 
- Website error and warning (canarytrace.console)
- Website error and warning donut (canarytrace.console) 

Error:
```
The index pattern c.console-* does not contain any of the following compatible field types: number, boolean, date, ip, or string
The field "context.keyword" associated with this object no longer exists in the index pattern. Please use another field.
```

2.) Canarytrace User-journey Overview dashboard

Error:
`No matching indices found: No indices match pattern "c.coverage-audit-*"`

Can you help me how to fill data in following index patterns:
c.console-*
c.coverage-audit

**Comment by rdpanek** [comment]
Hi, enable functions for collection of this data types. You must have CanarytracePro https://canarytrace.com/docs/guides/cli#services
